#Get the body data from mocap
uint8 id
---
QuadPosition pos
